packages/snaps-rpc-methods/src/restricted/getBip44Entropy.ts

```diff
@@ -108,12 +107,11 @@ export function getBip44EntropyImplementation({
 }: GetBip44EntropyMethodHooks) {
   return async function getBip44Entropy(
     args: RestrictedMethodOptions<GetBip44EntropyParams>,
-  ): Promise<JsonBIP44CoinTypeNode> {
+  ): Promise<GetBip44EntropyResult> {
     await getUnlockPromise(true);
 
     // `args.params` is validated by the decorator, so it's safe to assert here.
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const params = args.params!;
+    const params = args.params as GetBip44EntropyParams;
```

Comment:
Should this be asserted?

Reply:
As the comment explains, the decorator already checks if the params are set.

Reply:
Yeah, apparently I can't read